src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -549,21 +551,26 @@ private void InitializeHub(THub hub, HubConnectionContext connection)
 
     private static Task<bool> IsHubMethodAuthorized(IServiceProvider provider, HubConnectionContext hubConnectionContext, HubMethodDescriptor descriptor, object?[] hubMethodArguments, Hub hub)
     {
+        var endpoint = hubConnectionContext.Features.Get<IEndpointFeature>()?.Endpoint;
```

Comment:
So here was what I was asking about, how do I get the endpoint metadata out, this doesn't look correct because looking at the current auth tests, I don't think it even exposes anything like an HttpContext in the TestClient that the tests use, or at least other than stuffing in my own IEndpointFeature into the test client, it feels like I'm doing something wrong

Reply:
Ok, so there is no endpoint for a Hub method. The endpoint is already running at this point in the code. We just grab the metadata off the Hub method and use that. So if you look at `DiscoverHubMethods(...)` we grab the AuthorizeAttribute off of the methodInfo.

Reply:
So how does that work for these new policies, since they only exist as metadata off an Endpoint, there is no authorize attribute equivalent I can look at without the endpoint metadata?

Reply:
I guess it doesn't apply until we do the `[RequireClaims]` change?

Reply:
@davidfowl what do we want to do exactly for signalR here?

Reply:
Ok confirmed in standup that we don't have to do anything here since it won't work, so I will remove these changes in the dispatcher for now